src/main/java/org/broadinstitute/hellbender/tools/exome/ACSCopyRatioModeller.java

```diff
@@ -332,6 +332,8 @@ private CopyRatioDataCollection(final SegmentedModel segmentedModel) {
             //this may not be in genomic order, depending on how the segments are sorted in the segment file,
             //so we cannot simply take the list of coverages in the order from TargetCollection.targets()
             final List<Double> coverages = new ArrayList<>((int) targetCoverages.exomeSize());
+
+            //TODO: this might actually be more concise without streams
```

Comment:
This TODO was meant for a different branch, but since we're talking about it I went ahead and made it a little prettier using `flatMap`.